icons/attention.svg

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 46.2 (44496) - http://www.bohemiancoding.com/sketch -->
+    <title>alert / attention / dark</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="Icons" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+        <g id="alert-/-attention-/-dark">
+            <g id="attention">
+                <polygon id="Shape" points="0 0 24 0 24 24 0 24"></polygon>
```

Comment:
The current icon parsing script doesn't pick up the `<polygon>` element, this will need to be flattened in a design tool

Reply:
Chatted with Oliver about this stuff. I think this is an "artboard" left over from the design tool. The parse-icons script will just ignore this so I think it's fine if the icon renders correctly